src/ai/ai-cli.csproj

```diff
@@ -126,15 +126,16 @@
     <PackageReference Include="Azure.AI.OpenAI" Version="2.0.0-beta.2" />
   </ItemGroup>
 
-  <ItemGroup>
-    <ProjectReference Include="..\common\common.csproj" />
-    <ProjectReference Include="..\extensions\helper_functions_extension\helper_functions_extension.csproj" />
-    <ProjectReference Include="..\extensions\speech_extension\speech-extension.csproj" />
-    <ProjectReference Include="..\extensions\template_extension\template_extension.csproj" />
-    <ProjectReference Include="..\extensions\testframework\YamlTestFramework.csproj" />
-    <ProjectReference Include="..\telemetry\aria\telemetry.aria.csproj" />
-  </ItemGroup>
+	<ItemGroup>
+		<ProjectReference Include="..\common\common.csproj" />
```

Comment:
can you switch these back to "2 spaces per tab"? i think the whole file is that way, yeah?